src/core/InstancedBufferAttribute.js

```diff
@@ -1,40 +1,38 @@
 import { BufferAttribute } from './BufferAttribute.js';
 
-function InstancedBufferAttribute( array, itemSize, normalized, meshPerAttribute ) {
+class InstancedBufferAttribute extends BufferAttribute {
 
-	if ( typeof ( normalized ) === 'number' ) {
+	constructor( array, itemSize, normalized, meshPerAttribute ) {
 
-		meshPerAttribute = normalized;
+		super( array, itemSize, normalized );
 
-		normalized = false;
+		Object.defineProperty( this, 'isInstancedBufferAttribute', { value: true } );
 
-		console.error( 'THREE.InstancedBufferAttribute: The constructor now expects normalized as the third argument.' );
+		if ( typeof ( normalized ) === 'number' ) {
 
-	}
-
-	BufferAttribute.call( this, array, itemSize, normalized );
+			meshPerAttribute = normalized;
 
-	this.meshPerAttribute = meshPerAttribute || 1;
+			normalized = false;
 
-}
+			console.error( 'THREE.InstancedBufferAttribute: The constructor now expects normalized as the third argument.' );
 
-InstancedBufferAttribute.prototype = Object.assign( Object.create( BufferAttribute.prototype ), {
+		}
 
-	constructor: InstancedBufferAttribute,
+		this.meshPerAttribute = meshPerAttribute || 1;
```

Comment:
@Mugen87  Is there use default parameter?

Reply:
This should be a default parameter, too.